pkg/metrics/helpers.go

```diff
@@ -0,0 +1,33 @@
+package metrics
+
+import (
+	"bytes"
+	"strings"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/common/expfmt"
+)
+
+func FormatMetrics(cs ...prometheus.Collector) string {
```

Comment:
Could we please add some comments here, for next generation of UI/Go developers ? 😄